packages/react-core/src/components/Dropdown/DropdownToggleCheckbox.tsx

```diff
@@ -25,6 +28,10 @@ export interface DropdownToggleCheckboxProps
   id: string;
   /** Aria-label of the checkbox */
   'aria-label': string;
+  /** Text describing current loading status or progress */
+  defaultProgressAriaValueText?: string;
+  /** Aria-label for the default progress icon */
+  defaultProgressAriaLabel?: string;
```

Comment:
More of a nit, but for defaultProgressAriaLabel we could add on some verbiage similar to Spinner:

```suggestion
  /** Text describing current loading status or progress */
  defaultProgressAriaValueText?: string;
  /** Aria-label for the default progress icon to describe what is loading */
  defaultProgressAriaLabel?: string;
```

Additionally should these props also be beta?